### ISSUE TITLE
Add enable/disable tiles to plone.app.mosaic control panel

### ISSUE THREAD

**Issue by cguardia** [plone/plone.app.mosaic#360]
Some potentially useful tiles for advanced users, like the raw html, are disabled by default on install. Other tiles are sometimes not required for specific customers or can be confusing for some users.

A control panel that allows admins to enable or disable tiles could be useful in these cases. The insert menu would only show enabled tiles then. Is there any interest in having this? I could work on it if that's the case.

**Comment by datakurre** [comment]
Yes. A control panel has been long due. There was a such control panel in Deco, but it was removed in some Deco sprint where configuration was refactored into current format (by @jaroel, who may or may not remember the reasons). I recall that tile configuration does not have clear flag for enabling / disabling tiles from UI. Setting a non-existing category might work, but of course you could also implement a new setting for that.

**Comment by hvelarde** [comment]
FWIW, we do something like that in collective.cover and you both were involved.

**Comment by jaroel** [comment]
I think we favoured plone.app.registry over custom storage at that point. See also https://github.com/plone/plone.app.mosaic/issues/173#issuecomment-202782141

AFAIK we could easily add an `enabled` field to the registry for ITile, and check it in https://github.com/plone/plone.app.mosaic/blob/master/src/plone/app/mosaic/registry.py#L213 or https://github.com/plone/plone.app.mosaic/blob/master/src/plone/app/mosaic/registry.py#L314 .

Then a control panel should be easy; just use the plone.registry control panel features.
Doing so would also mean accessing the data through plone.restapi and plone.api would be super simple!
